http/http-advanced-reactive/src/main/resources/application.properties

```diff
@@ -42,6 +42,8 @@ quarkus.keycloak.policy-enforcer.paths.multipart-form-data.path=/api/multipart-f
 quarkus.keycloak.policy-enforcer.paths.multipart-form-data.enforcement-mode=DISABLED
 quarkus.keycloak.policy-enforcer.paths.hello.path=/api/hello/*
 quarkus.keycloak.policy-enforcer.paths.hello.enforcement-mode=DISABLED
+quarkus.keycloak.policy-enforcer.paths.greeting.path=/api/greeting/*
```

Comment:
I deprecated `path` property some times ago and implemented `paths` just to avoid what I see here. We keep adding new named configs why we can just add another path. Anyway, it is for a separate commit.

Reply:
see `quarkus.keycloak.policy-enforcer.paths."paths".paths` and you won't find `path` in the docs. anyway, there is no hurry, security super slow in removing properties. we have years I bet.